### PR TITLE
removing myself from CODEOWNERS

### PR DESCRIPTION
**Pull request by RCopeland**
## What does this PR do?
Removes me from the code owners file.

### CODE REVIEW

**Review by kkieninger — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -2,4 +2,4 @@
 # the repo. Unless a later match takes precedence,
 # @RCopeland and @afebbraro and @guern1kn will be requested for
```
Comment (guern1kn) [inline .github/CODEOWNERS:3]: Remove yourself here too.

**Review by guern1kn — CHANGES_REQUESTED** [review]
Left a comment.

File: .github/CODEOWNERS (hunk 1 above)
Comment (RCopeland) [reply .github/CODEOWNERS:3]: lol will do

**Review by afebbraro — CHANGES_REQUESTED** [review]
...

**Review by guern1kn — APPROVED** [review]
image

**Pull request merged**